src/igvfd/schemas/rodent_donor.json

```diff
@@ -53,11 +53,33 @@
                 "product_id"
             ],
             "comment": "Donor with lot_id must include product_id."
+        },
+        "individual_rodent": {
+            "comment": "A rodent identifier is required if the donor is an individual rodent but may not be submitted if it is not.",
+            "if": {
```

Comment:
I don't think the rodent identifier can be submitted if it is not an individual_rodent. I would remove the part of the sentence after the "but"